WTWSMS/events/religious_events.txt

```diff
@@ -3634,11 +3680,11 @@ character_event = {
 				trait = zealous
 			}
 			modifier = {
-				factor = 3
+				factor = 5
```

Comment:
Compare to the other values and it is accurately represented, given that it corresponds to other similar traits.